### PR TITLE
Require the installation of packages for debian/ubuntu added.

### PR DESCRIPTION
**Pull request by ky0ncheng**
Require the installation of packages for debian/ubuntu added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,7 +253,6 @@ cd ./..
 ./autogen.sh
 ./configure
```
Comment (kimw) [inline README.md]: This line should be `./configure --prefix=/usr`

Comment (ky0ncheng) [inline README.md]: fixed.

File: README.md [hunk 2]
```diff
@@ -247,12 +247,12 @@ git submodule update --init --recursive
...
 cd libsodium
 ./autogen.sh
-./configure
+./configure --prefix=/usr
 make && make install
 cd ./..
 ./
 ./autogen.sh
-./configure
+./configure --prefix=/usr
```
Comment (kimw) [inline README.md]: There, line 255, should NOT add `--prefix` on compiling ss-libev. Only the libsodium need it.

Comment (kimw) [inline README.md]: There, line 255, should NOT add `--prefix` on compiling ss-libev. Only the libsodium need it.

Comment (ky0ncheng) [inline README.md]: ok

File: README.md [hunk 3]
```diff
@@ -240,7 +240,20 @@ e.g. Ubuntu, Debian or Linux Mint, you can build the binary like this:
 
 ```bash
 # Debian / Ubuntu
-sudo apt-get install --no-install-recommends build-essential autoconf libtool libssl-dev libpcre3-dev asciidoc xmlto zlib1g-dev
+sudo apt-get install --no-install-recommends autoconf libtool asciidoc zlib1g-dev libssl-dev libpcre3 libpcre3-dev build-essential libpcre3 asciidoc libpcre3-dev zlib1g-dev libssl-dev automake libudns-dev libev-dev xmlto -y
```
Comment (rogers0) [inline README.md]: so many duplicates..

File: README.md [hunk 4]
```diff
@@ -240,7 +240,8 @@ e.g. Ubuntu, Debian or Linux Mint, you can build the binary like this:
 
 ```bash
 # Debian / Ubuntu
-sudo apt-get install --no-install-recommends autoconf libtool asciidoc zlib1g-dev libssl-dev libpcre3 libpcre3-dev build-essential libpcre3 asciidoc libpcre3-dev zlib1g-dev libssl-dev automake libudns-dev libev-dev xmlto -y
+sudo apt-get install --no-install-recommends autoconf libtool asciidoc zlib1g-dev libssl-dev libpcre3 libpcre3-dev build-essential libpcre3 asciidoc libpcre3-dev zlib1g-dev libssl-dev automake libudns-dev libev-dev xmlto
```
Comment (rogers0) [inline README.md]: don't remove "-y", please

Comment (ky0ncheng) [inline README.md]: This is step, not script. I need user to confirm it.

Comment (rogers0) [inline README.md]: By entering the command, user already requested to install those packages, and it's without recommends packages.
Why bother the extra confirmation?

File: README.md [hunk 5]
```diff
@@ -90,7 +90,7 @@ Shadowsocks-libev is available in the official repository for Debian 9("Stretch"
 
 ```bash
 sudo apt update
-sudo apt install shadowsocks-libev
+sudo apt install shadowsocks
```
Comment (ushuz) [inline README.md]: `shadowsocks` and `shadowsocks-libev` are different.

**Pull request closed without merging**